doc/modules/neighbors.rst

```diff
@@ -415,14 +415,14 @@ depends on a number of factors:
   a significant fraction of the total cost.  If very few query points
   will be required, brute force is better than a tree-based method.
 
-Currently, ``algorithm = 'auto'`` selects ``'brute'`` if :math:`k >= N/2`,
+Currently, ``algorithm = 'auto'`` selects ``'brute'`` if :math:`D > 15`,
 the input data is sparse, or ``effective_metric_`` isn't in
 the ``VALID_METRICS`` list for either ``'kd_tree'`` or ``'ball_tree'``.
 Otherwise, it selects the first out of ``'kd_tree'`` and ``'ball_tree'``
 that has ``effective_metric_`` in its ``VALID_METRICS`` list.
-This choice is based on the assumption that the number of query points is at
-least the same order as the number of training points, and that ``leaf_size``
-is close to its default value of ``30``.
+This choice is based on the assumption that the intrinsic dimensionality of the
+data is generally to high for tree-based methods — to provide significant
```

Comment:
is this a unicode dash?

Reply:
Yep (U+2014)

Reply:
We can't use unicode unfortunately as we still have a pdflatex build somewhere that doesn't support it.

Reply:
Alright, can replace it with a comma instead